src/pages/index.js

```diff
@@ -97,31 +159,168 @@ function getCardElement(data) {
     openModal(popupModal);
   });
 
-  cardLikeButton.addEventListener("click", () => {
-    cardLikeButton.classList.toggle("card__like-button_liked");
+  cardLikeButton.addEventListener("click", (evt) => {
+    handleLikeButton(evt, data._id);
   });
 
-  cardDeleteButton.addEventListener("click", () => {
-    cardElement.remove();
-  });
+  cardDeleteButton.addEventListener("click", () =>
+    handleDeleteCard(cardElement, data)
+  );
 
   return cardElement;
 }
 
+editAvatarButton.addEventListener("click", () => {
+  openModal(editAvatarModal);
+});
+
+editAvatarForm.addEventListener("submit", (evt) => handleAvatarSubmit(evt));
+
+/* function handleAvatarSubmit(evt) {
+  setButtonText(editAvatarSubmitButton, true);
+  api
+    .editAvatar(editAvatarInput.value)
+    .then((data) => {
+      profileAvatar.src = data.avatar;
+      evt.target.reset();
+      disableButton(editAvatarSubmitButton, settings);
+      closeModal(editAvatarModal);
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      setButtonText(editAvatarSubmitButton, false);
+    });
+}
+ */
+function handleAvatarSubmit(evt) {
+  function makeRequest() {
+    return api.editAvatar(editAvatarInput.value).then((data) => {
+      profileAvatar.src = data.avatar;
+      evt.target.reset();
```

Comment:
You have `evt.target.reset();` inside `handleSubmit`, so you don't need it here anymore